docs/developer-guide/quick-start-guide.md

```diff
@@ -0,0 +1,116 @@
+
+First, let's put some minimum viable HTML onto the page. This is not the HTML you would use in production, but for prototyping with the Build Service it's fine.
+
+
+	<html>
+		<head>
+			<!-- build service links and scripts here -->
+		</head>
+		<body>
+			Hello!
+			<button>Here's a button</button>
+		</body>
+	</html>
+
+
+
+## 2. Include the CSS
+
+OK, we've got a button there, but it's completely unstyled. Let's use the `o-buttons` Origami module to style it.
+
+We're going to use the Build Service to provide the styles.
+
+To do that we need to request the `o-buttons` CSS from the Build Service:
+
+```
+https://build.origami.ft.com/v2/bundles/css?modules=o-buttons
+```
+
+This request says "give me the CSS for o-buttons at the latest version"
+
+Let's add that to our page in the `<head>` as a `<link ...>` tag
+
+```
+<link rel="stylesheet" href="https://build.origami.ft.com/v2/bundles/css?modules=o-buttons">
+```
+
+<aside class='read-more'>
+<strong>Read more about component versioning</strong>
+<p>This example only uses the component tag (o-buttons). To use Origami modules in production code safely, you should also use a version number with your Build Service request.</p>
+<a class='o-buttons' href='{{site.baseurl}}/docs/developer-guide/modules/module-versioning/'>Find out more about component versioning</a>
+</aside>
+
+## 3. Add the classes
+
+Next add the right classes to your HTML. Some Origami components are more complicated than `o-buttons`. To help with this we provide example HTML already marked up with the right classes that you can copy into your project.
+
+For this tutorial though, you need to add the class `o-buttons` to the `<button>` tag.
+
+o-buttons has some design variations. Let's apply the `standout` variation by the adding `o-buttons--standout` class.
+
+```
+<button class="o-buttons o-buttons--standout">Here's a button</button>
+```
+
+## 4. Include the JavaScript
+
+`o-buttons` also has some JavaScript to make buttons work better on touch devices like phones. Getting JavaScript from the build service is as easy as getting the CSS.
+
+```
+https://build.origami.ft.com/v2/bundles/js?modules=o-buttons
+```
+
+Instead of using a `<link ...>` tag, use a `<src ...>` tag.
+
+```
+<src rel="stylesheet" href="https://build.origami.ft.com/v2/bundles/css?modules=o-buttons">
+```
+
+## 5. Putting it all together
+
+~~~ html
+<html>
+	<head>
+		<!-- build service links and scripts here -->
+		<src rel="stylesheet" href="https://build.origami.ft.com/v2/bundles/css?modules=o-buttons">
+		<link rel="stylesheet" href="https://build.origami.ft.com/v2/bundles/css?modules=o-buttons">
+	</head>
+	<body>
+		Hello!
+		<button class="o-buttons o-buttons--standout">Here's a button</button>
+	</body>
+</html>
+~~~
+
+## What's next?
+
+Well done! you've used an o-button on a page. Adding more components is very similar:
```

Comment:
ah ok good point, sorry I missed the distinction, just noticed all the other `o-things` were code highlighted below, but it makes sense not to in this case